### PR TITLE
{cae}[system/system] FLUENT v2021R2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3507.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/b3efcddaf5336ac20a52bd6fcf92912b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3114.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/825be729394d1a993e3274f393700e15 for a full test report.

File: easybuild/easyconfigs/f/FLUENT/FLUENT-2021R2.eb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name = 'FLUENT'
+version = '2021R2'
+
+homepage = 'http://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent'
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/f/FLUENT/FLUENT-2021R2.eb:4]:
```suggestion
homepage = 'https://www.ansys.com/Products/Simulation+Technology/Fluid+Dynamics/Fluid+Dynamics+Products/ANSYS+Fluent'
```

**Review by SebastianAchilles — COMMENTED** [review]
I think we can use `https` here.

File: easybuild/easyconfigs/f/FLUENT/FLUENT-2021R2.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/f/FLUENT/FLUENT-2021R2.eb:4]: https://www.ansys.com/products/fluids/ansys-fluent  seems to be more up to date

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2607.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/4b8b4b9f4b8b8629043dfe856f2b3ad1 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @boegel!

**Pull request merged**